http/install-common.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -e
+set -x
+
+ln -sf /usr/share/zoneinfo/UTC /etc/localtime
+sed -i -e 's/^#\(en_US.UTF-8\)/\1/' /etc/locale.gen
+locale-gen
+echo 'LANG=en_US.UTF-8' >/etc/locale.conf
+
+# setting the user credentials
+echo -e "${NEWUSER}\n${NEWUSER}" | passwd
+useradd -m -U vagrant
```

Comment:
You add a vagrant user in the common file, this is not needed for cloud builds (because we build no vagrant images there).

Reply:
I missed that line. Fixed